src/client/themes/classic/index.ts

```diff
@@ -54,7 +96,7 @@ const classicTheme: DefaultTheme = {
       background: 'transparent',
       borderColor: '#fff',
       color: '#fff',
-      disabledContrast: '0.1',
+      // disabledContrast: '0.1',
```

Comment:
Remove?

Reply:
I think we will need to reable it to define contrast type of the theme. But could be removed in the meantime, in the other pr I mentioned tho